packages/@lwc/wire-service/src/wiring.ts

```diff
@@ -253,7 +253,7 @@ export class WireEventTarget {
             });
             this._cmp.dispatchEvent(internalDomEvent);
             return false; // canceling signal since we don't want this to propagate
-        } else if (evt.type === 'WireContextEvent' || evt.type === 'wirecontextevent') {
+        } else if (evt.type === 'wirecontextevent') {
```

Comment:
are there tests for this?

Reply:
None that I found. It seems when the feature was implemented no tests were added.

Reply:
https://github.com/salesforce/lwc/tree/master/packages/integration-karma/test/context

Reply:
Those tests exercise the v2 context mechanism: `LinkContextEvent`. I believe this `WireContextEvent` is the legacy v1 implementation which still has one consumer owned by @jbenallen .

Reply:
got it.